index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+require('dotenv').config();
+// const app = require('./src/server.js');
+const { db } = require('./src/auth/models');
+const server = require('./src/server.js');
+
+const PORT = process.env.PORT || 3001;
+
+db.sync().then(() => {
+  server.start(PORT);
+});
```

Comment:
Only this db instance gets started, so the food and clothes models don't get created and can't be synced. That's why we need our db instance to be a singleton.